Explorer/Assets/DCL/SDKComponents/MediaStream/Systems/VideoEventsSystem.cs

```diff
@@ -45,9 +47,11 @@ private void PropagateVideoEvents(ref CRDTEntity sdkEntity, ref MediaPlayerCompo
         {
             if (!frameTimeBudget.TrySpendBudget()) return;
 
-            VideoState newState = GetCurrentVideoState(mediaPlayer.MediaPlayer.Control);
+            VideoState newState = GetCurrentVideoState(mediaPlayer.MediaPlayer.Control, mediaPlayer.PreviousPlayingTimeCheck, mediaPlayer.LastStateChangeTime);
 
             if (mediaPlayer.State == newState) return;
+            mediaPlayer.LastStateChangeTime = Time.timeSinceLevelLoad;
```

Comment:
not sure, but  maybe better to use `Time.realtimeSinceStartup` just in  some theory case of loading different non-additive scenes in future (highly un-likely, but still). Decide on your own :)

Reply:
tackled at https://github.com/decentraland/unity-explorer/pull/2474/commits/94162ae6cd1c59aa17ce5dbf1902129a7aa8405f